test/fuzz/README.md

```diff
@@ -0,0 +1,56 @@
+# Fuzzing
+
+ Fuzzing or fuzz testing is an automated software testing technique that
+ involves providing invalid, unexpected, or random data as inputs to a
+ computer program. A fuzzing target is defined for running the tests.
+ These targets are defined in the test/fuzz directory and run using
+ the instructions given below:
+
+- Install the package from <https://github.com/mdempsky/go114-fuzz-build>
+  using go get command. This is the link for the fuzzing tool that uses
+  libfuzzer for its implementation purposes. I set it up and made a sample
+  function to run the fuzzing tests. In these steps following are the important steps
+
+   a.  Run below command to build the file for the function by giving its location.
+
+    ```bash
+    mkdir output
+    cd output
+    go114-fuzz-build -o yaml_FuzzNewIPPoolManager.a -func FuzzNewIPPoolManager ../test/fuzz/
+    ```
+
+    b. Run below command to make C binary files for the same function.
+
+    ```
+    clang -o yaml_FuzzNewIPPoolManager  yaml_FuzzNewIPPoolManager.a -fsanitize=fuzzer
+    ```
+
+    c. Run the fuzzer as below
+
+    ```
+    ./yaml_FuzzNewIPPoolManager
+    ```
+
+- Execution: The execution of the fuzzer for my sample code kept on
+  running with below sample outputs:
```

Comment:
Avoid using subjective phrases ("my sample code"). Instead, something like "Here is an example of test outputs:"
Move the explanation part to below the output btw.

Reply:
resolved this issue.